packages/core/src/util/chain.ts

```diff
@@ -0,0 +1,49 @@
+import { RPC, config, utils } from '@ckb-lumos/lumos'
+
+// Special cells in genesis transactions: (transaction-index, output-index)
+const SpecialCellLocation: {
```

Comment:
Better to add a reference of this locations

Reply:
I'm not quite sure where this reference can be found, these magic location is what I see in the source code of ckb-cli

Reply:
Please refer to the code of ckb-cil, and I'll open an issue in ckb-cli repo for detailed info

Reply:
https://github.com/nervosnetwork/ckb-sdk-rust/blob/0d6cbb38f59d6bf21982157874c17a1e71b83c81/src/constants.rs#L20-L24

Reply:
My bad, I didn't notice the comment above, which clearly implies the definition of `[number, number]` is `[transaction-index, output-index]`

Among them, `SIGNHASH_GROUP`, `MULTISIG_GROUP`, `DAO` can be found at https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0024-ckb-genesis-script-list/0024-ckb-genesis-script-list.md
`SIGHASH`, `MULTISIG` are not used normally. I'm not sure if these two will be used in our project.

Besides, we can use labeled tuple for readability

Ref: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-0.html#labeled-tuple-elements